src/v/storage/log_manager.cc

```diff
@@ -507,7 +513,10 @@ ss::future<> log_manager::recover_log_state(const ntp_config& cfg) {
       });
 }
 
-ss::future<ss::shared_ptr<log>> log_manager::do_manage(ntp_config cfg) {
+ss::future<ss::shared_ptr<log>> log_manager::do_manage(
+  ntp_config cfg,
+  raft::group_id group,
```

Comment:
would be great to have an assertion that if `translator_batch_types` is non-empty, then `group` has value

Reply:
Done